AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -183,12 +185,16 @@ + (void)signOut {
 }
 
 - (void)signIn {
+  if ([[MS_Reachability reachabilityForInternetConnection] currentReachabilityStatus] == NotReachable) {
```

Comment:
Is it safe to be called from background thread? We had threading issue in Reachability so please test.

Reply:
Yes. There are no locks there.